app/src/main/java/unipd/se18/ocrcamera/inci/Inci.java

```diff
@@ -0,0 +1,81 @@
+package unipd.se18.ocrcamera.inci;
+
+import android.util.Log;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.util.ArrayList;
+
+import com.opencsv.CSVReader;
+
+/**
+ * Class for Inci database management
+ * The database is in CSV format, using openCSV to parse the file.
+ * For more informations about openCSV:
+ * http://opencsv.sourceforge.net/
+ * @author Francesco Pham
+ */
+public class Inci {
+
+    private static final String TAG = "Inci";
+
+    private static final int COL_COSING_REF_NO = 0;
+    private static final int COL_INCI_NAME = 1;
+    private static final int COL_DESCRIPTION = 6;
+    private static final int COL_FUNCTION = 8;
```

Comment:
Enums cannot be used in this case because they are just labels and values cannot be set. Here I needed to specify the column numbers in which those values are found in the CSV. Maybe I should have put a comment to be more clear on the functions of those constants, sorry.